File: src/main/java/plus/maa/backend/controller/request/user/SendRegistrationTokenDTO.java
package plus.maa.backend.controller.request.user;

import jakarta.validation.constraints.Email;
import jakarta.validation.constraints.NotBlank;
import lombok.Data;

@Data
public class SendRegistrationTokenDTO {
    @NotBlank(message = "邮箱格式错误")
    @Email(message = "邮箱格式错误")
    private String email;
}

File: src/main/kotlin/plus/maa/backend/controller/request/user/SendRegistrationTokenDTO.kt
package plus.maa.backend.controller.request.user

import jakarta.validation.constraints.Email
import jakarta.validation.constraints.NotBlank

data class SendRegistrationTokenDTO(
    @field:NotBlank(message = "邮箱格式错误")
    @field:Email(message = "邮箱格式错误")
    val email: String,
)
